internal/db/schema/migrations/oss/postgres/42/02_add_job_config_table.up.sql

```diff
@@ -0,0 +1,22 @@
+begin;
+
+    create table job_config (
```

Comment:
The table is missing a primary key

Reply:
Yeah I should probably just slap a `primary key (job_plugin_id, job_name)` to replace the `unique` below. Good note for the future implementation.

Reply:
Fixed